creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -419,6 +419,25 @@ an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to dissable bridge interface with pod network.
+This is done with a designated flag.
+The default value of the flag is 'true', meaning bridge interface is enabled.
+to disable it change flag value to false. For example:
```

Comment:
this line should start with upper case

Reply:
you single-quote 'true' above but not false here. Please make the text consistent.